src/context.ts

```diff
@@ -23,7 +23,17 @@ import * as github from '@actions/github';
 import {GitHub} from './github';
 
 export class Context {
-  private static readonly _tmpDir = fs.mkdtempSync(path.join(os.tmpdir(), 'docker-actions-toolkit-'));
+  private static readonly _tmpDir = fs.mkdtempSync(path.join(Context._tryCreateDirctory(process.env.RUNNER_TEMP) || os.tmpdir(), 'docker-actions-toolkit-'));
+
+  private static _tryCreateDirctory(directoryPath: string | undefined): string | undefined {
+    if (directoryPath !== undefined) {
+      const createdPath = fs.mkdirSync(directoryPath, {recursive: true});
+      if (createdPath !== directoryPath) {
+        return undefined;
+      }
+    }
+    return directoryPath;
+  }
```

Comment:
Don't think `_tryCreateDirctory` is necessary, can just be:

```suggestion
  private static readonly _tmpDir = fs.mkdtempSync(path.join(process.env.RUNNER_TEMP || os.tmpdir(), 'docker-actions-toolkit-'));
```

Reply:
A directory at path `RUNNER_TEMP` may not exist, in which case `fs.mkdtempSync(path.join(process.env.RUNNER_TEMP, 'docker-actions-toolkit-'))` would throw an exception. Even though the provided string is a prefix, the directory in which the temporary directory would be created must already exist.

Reply:
Runner worker makes sure temp folder exists from what I see: https://github.com/actions/runner/blob/a4c57f27477077e57545af79851551ff7f5632bd/src/Runner.Worker/TempDirectoryManager.cs#L43-L44

Reply:
Ahh good point. This means the tests also need to point `RUNNER_TEMP` to an existing directory tough.
Will update the PR.

Reply:
> This means the tests also need to point `RUNNER_TEMP` to an existing directory tough.
> Will update the PR.

This is already set in jest config: https://github.com/docker/actions-toolkit/blob/b525cd9de82f6ef6f521081921cac18cb43e04b9/jest.config.ts#L26